templates/includes/sidebar.html

```diff
@@ -1,4 +1,22 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>
+      User:
+      <a href="{% url "taxi:driver-detail" pk=user.id %}">{{ user.username }}</a>
```

Comment:
Ensure that the `user.id` is the correct identifier for the `driver-detail` view. If `user` is not directly related to the `Driver` model, this might cause an issue.